specs/phase0/p2p-interface.md

```diff
@@ -333,6 +333,7 @@ The following validations MUST pass before forwarding the `signed_beacon_block`
   i.e. validate that `signed_beacon_block.message.slot > compute_start_slot_at_epoch(state.finalized_checkpoint.epoch)`
   (a client MAY choose to validate and store such blocks for additional purposes -- e.g. slashing detection, archive nodes, etc).
 - _[IGNORE]_ The block is the first block with valid signature received for the proposer for the slot, `signed_beacon_block.message.slot`.
+- _[IGNORE]_ The block is not slashable.
```

Comment:
I don't quite understand this one. How can the block be slashable while it's the first block with valid signature received for the slot.